version/version.go

```diff
@@ -34,7 +34,12 @@ func computeGenCodeCompabilityRange() (r genCodeCompatbilityRangeHolder) {
 	r.begin = parseSemVerOrPanic(Version)
 	r.begin.Patch = 0
 	r.end = r.begin
-	r.end.Minor++
+	if r.begin.Major == 0 {
+		r.end.Minor++
+	} else {
+		r.end.Minor = 0
```

Comment:
Don't we want to flip `begin.Minor` to 0 too? Otherwise we're enforcing the latest ('current') version of thriftrw as the minimum version for code that was generated earlier.